src/snowflake/connector/aio/_network.py

```diff
@@ -827,6 +827,11 @@ async def _request_exec(
             RuntimeError,
             AttributeError,  # json decoding error
         ) as err:
+            if "Event loop is closed" in str(err):
+                logger.warning(
```

Comment:
do you figure it out what session delete does? if it's some critical op that I think this could be a logging.INFO
```suggestion
                logger.info(
```